.gitignore

```diff
@@ -31,3 +31,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# local env variables
+.env.local
```

Comment:
We should be using `.env` as the filename for the local env variables according to the instructions.